django/thunderstore/social/api/experimental/views/current_user.py

```diff
@@ -150,8 +160,10 @@ def get_social_auth_connections(user: UserType) -> List[SocialAuthConnection]:
     return [
         {
             "provider": sa.provider,
-            "username": sa.extra_data[OAUTH_USERNAME_FIELDS[sa.provider]],
-            "avatar": sa.extra_data.get(OAUTH_AVATAR_FIELDS[sa.provider]),
+            "username": sa.extra_data.get(OAUTH_USERNAME_FIELDS.get(sa.provider)),
+            "avatar": sa.extra_data.get(OAUTH_AVATAR_FIELDS[sa.provider])
```

Comment:
I think that should functionally be equivalent yeah and I'm no longer sure why I changed it to this (I had it originally as you suggested). Keeping it as is since this could be refactored a bit more if I were to refactor it, and I don't want to head down that rabbit hole now.